WasatchNET/EEPROMJSON.cs

```diff
@@ -0,0 +1,425 @@
+        public int StartupIntTimeMS;
+        public int StartupTempC;
+        public int StartupTriggerMode;
+        public double DetectorGain;
+        public double DetectorGainOdd;
+        public int DetectorOffset;
+        public int DetectorOffsetOdd;
+        public double[] WavecalCoeffs;
+        public double[] TempToDACCoeffs;
+        public double[] ADCToTempCoeffs;
+        public double[] LinearityCoeffs;
+        public int DetectorTempMax;
+        public int DetectorTempMin;
+        public int ThermistorBeta;
+        public int ThermistorResAt298K;
+        public string CalibrationDate;
+        public string CalibrationBy;
+        public string DetectorName;
+        public int ActualPixelsHoriz;
+        public int ActivePixelsHoriz;
+        public int ActivePixelsVert;
+        public int MinIntegrationTimeMS;
+        public int MaxIntegrationTimeMS;
+        public int ROIHorizStart;
+        public int ROIHorizEnd;
+        public int[] ROIVertRegionStarts;
+        public int[] ROIVertRegionEnds;
+        public double[] LaserPowerCoeffs;
+        public double MaxLaserPowerMW;
+        public double MinLaserPowerMW;
+        public double ExcitationWavelengthNM;
+        public double AvgResolution;
+        public int[] BadPixels;
+        public string UserText;
+        public string ProductConfig;
+        public int RelIntCorrOrder;
+        public double[] RelIntCorrCoeffs;
+        public bool Bin2x2;
+        public bool FlipXAxis;
+
+        public override bool Equals(object obj)
+        {
+            var item = obj as EEPROMJSON;
+
+            //public string Serial;
+            if (item.Serial != this.Serial)
+                return false;
+            //public string Model;
+            if (item.Model != this.Model)
+                return false;
+            //public int SlitWidth;
+            if (item.SlitWidth != this.SlitWidth)
+                return false;
+            //public double BaudRate;
+            if (Math.Abs(item.BaudRate - this.BaudRate) > 0.00001)
+                return false;
+            //public bool IncBattery;
+            if (item.IncBattery != this.IncBattery)
+                return false;
+            //public bool IncCooling;
+            if (item.IncCooling != this.IncCooling)
+                return false;
+            //public bool IncLaser;
+            if (item.IncLaser != this.IncLaser)
+                return false;
+            //public int StartupIntTimeMS;
+            if (item.StartupIntTimeMS != this.StartupIntTimeMS)
+                return false;
+            //public int StartupTempC;
+            if (item.StartupTempC != this.StartupTempC)
+                return false;
+            //public int StartupTriggerMode;
+            if (item.StartupTriggerMode != this.StartupTriggerMode)
+                return false;
+            //public double DetectorGain;
+            if (Math.Abs(item.DetectorGain - this.DetectorGain) > 0.00001)
```

Comment:
seems like it would be making a bool floatEq(float a, float b, float thresh = 0.00001) function

And deleting extra comments